types/src/primary.rs

```diff
@@ -541,6 +541,16 @@ pub enum PrimaryMessage {
         from: PublicKey,
     },
 
+    CertificatesRoundsRequest {
+        limit: (Round, Round),
+        author: PublicKey,
+        requestor: PublicKey,
+    },
+    CertificatesRoundsResponse {
+        certificates: Vec<(CertificateDigest, Option<Certificate>)>,
+        from: PublicKey,
+    },
+
```

Comment:
Based on those types:
- is the `author` necessary?
- Nit: rename `limit` to `range`?
- for the Response, do we need the `Certificates`?

Reply:
Updated.